addons/actions.md

```diff
@@ -23,62 +23,88 @@ The following Actions are a core part of openHAB and do not need to be separatel
-* ` say(String text, String voice, String device)`: Says the given text through the given voice to the given output device (Mac OSX only).
+- `setMasterVolume(float volume)`: Sets the volume of the host machine (volume in range 0-1)
+- `increaseMasterVolume(float percent)`: Increases the volume by the given percent
+- `decreaseMasterVolume(float percent)`: Decreases the volume by the given percent
+- `float getMasterVolume()`: Returns the current volume as a float between 0 and 1
+- `playSound(String filename)`: Plays the given sound file. The file must be an mp3 or wav and located in `${openhab.home}/sounds`
+- `playStream(String url)`: Plays the audio stream at the given url
+- `say(String text)`: Says the given text through Text-to-Speech
+- `say(String text, String voice)`: Says the given text through the given voice (depends on the TTS engine and voices installed)
+- ` say(String text, String voice, String device)`: Says the given text through the given voice to the given output device (Mac OSX only).
 
 To get a list of available voices use `say -v ?` and to get a list of devices uses `say -a ?`.
 
 ### Logging Actions
 
-* `logDebug(String loggername, String logText)`: Logs logText on level DEBUG to openhab.log
-* `logInfo(String loggername, String logText)`: Logs logText on level INFO to openhab.log
-* `logWarn(String loggername, String logText)`: Logs logText on level WARN to openhab.log
-* `logError(String loggername, String logText)`: Logs logText on level ERROR to openhab.log
+- `logDebug(String loggername, String logText)`: Logs logText on level DEBUG to openhab.log
+- `logInfo(String loggername, String logText)`: Logs logText on level INFO to openhab.log
+- `logWarn(String loggername, String logText)`: Logs logText on level WARN to openhab.log
+- `logError(String loggername, String logText)`: Logs logText on level ERROR to openhab.log
 
 `logText` can be a compete String, constructed through concatination, or through arguments.
 
-* **Complete String Example:** `logInfo("Garage", "This is a complete String")`
-* **Concatination Example:** `logDebug("Lighting", "This is a string concatination:" + Light.name)`
-* **Arguments Example:** `logWarn("Alarm", "The {} window has been open for {} hours!", Window.name, timeOpen)`
+- **Complete String Example:** `logInfo("Garage", "This is a complete String")`
+- **Concatination Example:** `logDebug("Lighting", "This is a string concatination:" + Light.name)`
+- **Arguments Example:** `logWarn("Alarm", "The {} window has been open for {} hours!", Window.name, timeOpen)`
 
 One can configure whether specific log entries are logged out and where they get logged to (e.g. to a separate file) by [editing the logger configuration]({{base}}/administration/logging.html).
 
+### Notification actions
+
+Notification actions may be placed in `rules` to send alerts to mobile devices registered with <https://myopenhab.org>.  Three different actions are available:
```

Comment:
Consider Rules instead of `rules`
Replace `<https://myopenhab.org>` with `an [openHAB Cloud instance](https://github.com/openhab/openhab-cloud) like [myopenHAB.org](https://myopenhab.org)`

The exact wording is up to you ;)

Reply:
To be more in line with the other sections maybe move this below the list?

Reply:
Agree.  Will change in next commit.

Reply:
@bgilmer77 do you think "alert" is the same as notification? Do you think users could interpret alert as something scary?